Makefile

```diff
@@ -56,6 +56,10 @@ clean:
 nopyc:
 	find . -name \*.pyc -delete
 
+prettier:
+	. venv/bin/activate; black twilio/rest
```

Comment:
Should this run on `twilio/rest` or on the repo root (which librarian is doing)?